### PR TITLE
Add my talk description for Nov 2

### PR DESCRIPTION
**Pull request by kellyk**
**Feel free to delete this message before submitting your PR.**

If you're submitting a PR for a talk or a performance, here's the format you should follow:

```yml
- 7:45 PM:
    type: Talk
    emoji: 📞
    title: |
      Your Title Here
    description: |
      Your description here.
    person:
      name: Foo Bar
      twitter: foobar
```

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -359,7 +359,11 @@
     - 9:00 PM:
         type: Talk
         emoji: 🤔
-        title: TBD
+        title: Web Animations
+        description: With so many different ways to animate the web, it can be hard
```
Comment (billyroh) [inline models/calendar.yml]: This needs to be like:
```yml
description: |
  With so many...
```
to be rendered correctly 😄

File: models/calendar.yml [hunk 2]
```diff
@@ -359,7 +359,12 @@
     - 9:00 PM:
         type: Talk
         emoji: 🤔
-        title: TBD
+        title: Web Animations
+        description: |
+        With so many different ways to animate the web, it can be hard
```
Comment (billyroh) [inline models/calendar.yml]: This text block should be indented one level in

Comment (billyroh) [inline models/calendar.yml]: If it helps, you can clone the repo to your local machine and run it with `npm start` :)

Comment (kellyk) [inline models/calendar.yml]: I have the repo cloned but I can't get it to show any content. I'm seeing a bunch of errors in the console like "Uncaught Error: Cannot find module "./index.md"". Anyway hope this fixed it :(

**Pull request merged**